CMakeLists.txt

```diff
@@ -194,6 +243,40 @@ if(${TRITON_ENABLE_ONNXRUNTIME_OPENVINO})
   )
 endif() # TRITON_ENABLE_ONNXRUNTIME_OPENVINO
 
+#
+# ONNX Runtime - if requested, build ORT using docker
+#
+if(NOT ("${TRITON_BUILD_ONNXRUNTIME_LIBNAME}" STREQUAL ""))
```

Comment:
Should check for TRITON_BUILD_ONNXRUNTIME_VERSION instead of TRITON_BUILD_ONNXRUNTIME_LIBNAME since TRITON_BUILD_ONNXRUNTIME_LIBNAME will be  "libonnxruntime.so." if TRITON_BUILD_ONNXRUNTIME_VERSION is "" so TRITON_BUILD_ONNXRUNTIME_LIBNAME is never ""